### ISSUE TITLE
Extension Error: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by fxuniverse** [GoogleChrome/lighthouse#1213]
**Lighthouse Version**: 1.3.0
**Chrome Version**: 57.0.2950.4
**Error Message**: Failed to construct 'URL': Invalid URL
**Stack Trace**:
 ```TypeError: Failed to construct 'URL': Invalid URL
    at artifacts.DateNowUse.usage.filter.err (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1046:39)
    at Array.filter (native)
    at Function.audit (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1045:48)
    at Promise.resolve.then._ (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11365:20)```

**Comment by wardpeet** [comment]
@fxuniverse could you provide us with an url please.
do you have any chrome plugins installed? Do you have any options deselected on the extension?

// dupe of #1194

**Comment by fxuniverse** [comment]
https://cc3test.careercruising.com?debug=1
I do have many plugins, will try disabling
deselected: offline/flaky, progressively enhance, add to homescreen, splashscreen, matching colors, modern offline features.
basically I just need the first meaningful paint score

**Comment by wardpeet** [comment]
@fxuniverse could you also provide me with a timezone perhaps and time of trying in local time.

**Comment by wardpeet** [comment]
also is it the login page you are testing?

**Comment by fxuniverse** [comment]
yes, want to start from login as majority of assets are loaded then. timezone is EST, tried at 11:10

**Comment by wardpeet** [comment]
Thanks for the feedback, error should be fixed in #1218
